travis.gemspec

```diff
@@ -299,7 +299,9 @@ Gem::Specification.new do |s|
   s.add_dependency "launchy",               "~> 2.1"
   s.add_dependency "typhoeus",              "~> 0.6", ">= 0.6.8"
   s.add_dependency "pusher-client",         "~> 0.4"
-  s.add_dependency "addressable",           "~> 2.3"
+  if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("1.9.0")
```

Comment:
That's already taken care of on the `master` branch. You'd have to update this PR.